contexts/DonationContext/src/Domain/Model/DonationStatusMapper.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\DonationContext\Domain\Model;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class DonationStatusMapper {
```

Comment:
If I understand correctly this thing maps the Donation status we use in our domain model to a presentation specific status.

Two issues:

1. The Domain model really should not contain anything presentation specific. What made you not simply put this in the presenter? (Not saying it should be there, but it's the place I'd start with considering)

2. It is weird this mapper thing defines the constants for values in the domain model. Don't we already have those defined as constants somewhere? If not, having them in `Donation` or some `DonationStatus` thing seems much more appropriate.